transmogrifier/sources/xml/dspace_dim.py

```diff
@@ -224,59 +229,58 @@ def get_optional_fields(self, source_record: Tag) -> dict | None:
             )
 
         # summary, uses description list retrieved for notes field
-        for description in [
-            d for d in descriptions if d.get("qualifier") == "abstract" and d.string
-        ]:
-            fields.setdefault("summary", []).append(description.string)
+        # for description in [
+        #     d for d in descriptions if d.get("qualifier") == "abstract" and d.string
+        # ]:
+        #     fields.setdefault("summary", []).append(description.string)
+        fields["summary"] = self.get_summary()
 
         return fields
 
     @classmethod
-    def get_contents(cls, source_record: Tag) -> list[str]:
-        return [
-            str(contents.string)
-            for contents in source_record.find_all(
-                "dim:field",
-                element="description",
-                qualifier="tableofcontents",
-                string=True,
-            )
-        ]
+    def get_contents(cls, source_record: etree._Element) -> list[str]:
+        contents = cls.string_list_from_xpath(
+            source_record,
+            ".//dim:field[@element='description' and @qualifier='tableofcontents']",
+        )
+        return contents
 
     @classmethod
-    def get_dates(cls, source_record: Tag, source_record_id: str) -> list[timdex.Date]:
+    def get_dates(
+        cls, source_record: etree._Element, source_record_id: str
+    ) -> list[timdex.Date]:
         dates = []
         dates.extend(list(cls._parse_date_elements(source_record, source_record_id)))
         dates.extend(list(cls._parse_coverage_elements(source_record, source_record_id)))
         return dates
 
     @classmethod
     def _parse_date_elements(
-        cls, source_record: Tag, source_record_id: str
+        cls, source_record: etree._Element, source_record_id: str
     ) -> Iterator[timdex.Date]:
-        for date_element in source_record.find_all(
-            "dim:field", element="date", string=True
+        for date_element in cls.xpath_query(
+            source_record, ".//dim:field[@element='date']"
```

Comment:
Noting here, but think it's true throughout: probably don't need `.//` prefixing all Xpath queries.  If the `.xpath()` is called from the root element, then `//` would be sufficient to find an element anywhere in the XML file, as all elements are children of this.

Fairly minor, but felt worth mentioning.